apps/web/package.json

```diff
@@ -139,6 +140,7 @@
     "lodash": "~4.17.21",
     "lru-cache": "^4.1.1",
     "lucide-react": "^0.453.0",
+    "luxon": "^3.5.0",
```

Comment:
im a little hesitant to add/use Luxon until we are fully replacing moment with it, otherwise we have both filling up our bundle size. is it hard to use moment here for now?